sign.sh

```diff
@@ -71,7 +71,10 @@ signRelease()
       signToolPath=${signToolPath:-"/cygdrive/c/Program Files (x86)/Windows Kits/10/bin/10.0.17763.0/x64/signtool.exe"}
 
       # Sign .exe files
-      FILES=$(find . -type f -name '*.exe' -o -name '*.dll')
+      # The ./bin/C directory contains FIPS binaries that are signed by other means.
+      # Skip all files within this directory as signing them would cause failures when
+      # the libraries attempt to self verify themselves.
+      FILES=$(find . -type f -name '*.exe' -o -name '*.dll' -not -path './bin/C/*')
```

Comment:
Parentheses are needed around the `-o` subexpression:
```
      FILES=$(find . -type f '(' -name '*.exe' -o -name '*.dll' ')' -not -path './bin/C/*')
```
To be fair, they were missing before. :disappointed:

Reply:
Hello, I did run this find command on a windows machine and I got the result that I expected, just the single `.dll` file was removed from the bin/C directory search results. Are you seeing a different result?

It could be true that if the order of the .exe and .dll expressions were changed then this might not work correctly.

Reply:
It may not make a difference with the set of expected file and directory names, but it's still wrong.
Implicitly predicates are joined by `-a` (and) which has higher precedence than `-o` (or).
Making those operators explicit may make the issue clearer:
```
find .  '(' -type f -a -name '*.exe' ')' -o '(' -name '*.dll' -a -not -path './bin/C/*' ')'
```
A perhaps real problem would be that it **_will_** include `*.exe` files in the `bin/C` directory which isn't the intent as I understand it.